handbook/engineering/code-insights/index.md

```diff
@@ -31,8 +31,8 @@ The code insights team also keeps a [backlog GitHub project board](https://githu
 
 ## Members
 
-- [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) Product Manager
-- [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her) ([Product Designer](../../product/roles/product_designer.md))
+- [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him), [Product Manager](../../product/roles/index.md#product-manager)
+- [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her) ([Product Designer](../../product/roles/index.md#product-designer))
 - [Felix Becker](../../../company/team/index.md#felix-becker) ([Engineering Manager](../roles.md#engineering-manager)) {#code-insights-eng}
```

Comment:
```suggestion
- [Felix Becker](../../../company/team/index.md#felix-becker), [Engineering Manager](../roles.md#engineering-manager)
```
removed anchor

Reply:
I believe this anchor is used for the engineering org page or the org chart.